src/main/java/net/dries007/tfc/common/blockentities/rotation/WindmillBlockEntity.java

```diff
@@ -116,7 +116,7 @@ public WindmillBlockEntity(BlockEntityType<?> type, BlockPos pos, BlockState sta
         final Direction.Axis axis = state.getValue(WindmillBlock.AXIS);
 
         this.invalid = false;
-        this.node = new SourceNode(pos, Node.ofAxis(axis), Direction.fromAxisAndDirection(axis, Direction.AxisDirection.POSITIVE), 0f) {
```

Comment:
Can this be done without changing the internal rotation? Just for better compat with existing setups, the only change should be visual here, I don't think this should be necessary if we're just flipping the model,

Reply:
Yeah, I can do that instead